Content.Server/GameObjects/Components/Fluids/SpillHelper.cs

```diff
@@ -0,0 +1,98 @@
+using Content.Shared.Chemistry;
+using Robust.Server.Interfaces.GameObjects;
+
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Map;
+using Robust.Shared.IoC;
+using Robust.Shared.Map;
+
+namespace Content.Server.GameObjects.Components.Fluids
+{
+    public static class SpillHelper
+    {
+#pragma warning disable 649
+        [Dependency] private static IEntityManager _entityManager;
```

Comment:
These definitely are better off not being stored in a static. It'll probably provide issues with integration tests and such.

Just resolve them every time in the method, I don't think performance is gonna be a problem any time soon.